group/models.py

```diff
@@ -520,13 +513,6 @@ class GroupReportsPage(BasePage):
 
     subpage_types = ['base.IntranetPlainPage']
 
-    def clean(self):
-        """
-        Make sure page titles adhere to strict
-        formatting policy.
-        """
-        enforce_name_as_year(self.title)
-
```

Comment:
As I can tell both the `enforce_name_as_year` and `enforce_name_as_reports` are no longer used anywhere, so these can be removed from this file. That means deleting lines 53-77.

Reply:
`enforce_name_as_year` is still being used by the `GroupMeetingMinutesPage` class as I thought it made sense to retain that convention.